src/Editor/index.js

```diff
@@ -26,11 +26,8 @@ export class Editor extends React.Component {
         showMentions: PropTypes.bool,
         onHideMentions: PropTypes.func,
         editorStyles: PropTypes.object, 
-        placeholder: PropTypes.placeholder,
-        renderMentionList: PropTypes.oneOfType([
-            PropTypes.func,
-            PropTypes.null,
-        ]),
+        placeholder: PropTypes.string,
```

Comment:
Thank you for pointing this out. I am going to merge your pull request. However I will revert `renderMentionList` change as this will help to use default `mentionList template` for those peoples who just wants to test this plugin.